pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/UnsafeUtils.java

```diff
@@ -0,0 +1,295 @@
+package com.hedera.pbj.runtime.io.buffer;
+
+import java.lang.reflect.Field;
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.security.AccessController;
+import java.security.PrivilegedExceptionAction;
+
+final class UnsafeUtils {
```

Comment:
Lots of useful stuff in this class but feels like we should not be adding till needed. I do not see any code changes to use this new class from existing code.

Reply:
The ReadWriteDirectUnsafeByteBuffer subclass class in the BufferedData.java (the last one) is using this class. It is utilized in Direct ByteBuffer only, since there is no access to the byte buffer. It is being used by the test and it is working properly for new.